kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -0,0 +1,148 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+// System includes
+#include <limits>
+
+/* External includes */
+
+/* Project includes */
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "geometries/point_3d.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "spaces/ublas_space.h"
+#include "tests/cpp_tests/auxiliar_files_for_cpp_unnitest/test_element.h"
+#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h"
+#include "solving_strategies/strategies/explicit_solving_strategy_runge_kutta_4.h"
+
+namespace Kratos
+{
+namespace Testing
+{
+    typedef Node<3> NodeType;
+    typedef Geometry<NodeType> GeometryType;
+    typedef UblasSpace<double, Matrix, Vector> LocalSpaceType;
+    typedef UblasSpace<double, CompressedMatrix, Vector> SparseSpaceType;
+    typedef ExplicitBuilder< SparseSpaceType, LocalSpaceType > ExplicitBuilderType;
+    typedef ExplicitSolvingStrategyRungeKutta4<SparseSpaceType, LocalSpaceType> ExplicitSolvingStrategyRK4Type;
+
+    class AuxiliaryExplicitStrategiesTestElement : public Element
+    {
+    public:
+        KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(AuxiliaryExplicitStrategiesTestElement);
+
+        AuxiliaryExplicitStrategiesTestElement() = default;
+
+        AuxiliaryExplicitStrategiesTestElement(
+            IndexType NewId,
+            GeometryType::Pointer pGeometry) : Element(NewId, pGeometry) {};
+
+        AuxiliaryExplicitStrategiesTestElement(AuxiliaryExplicitStrategiesTestElement const &rOther) = delete;
+
+        void GetDofList(
+            DofsVectorType& rElementalDofList,
+            const ProcessInfo& rCurrentProcessInfo) const override
+        {
+            if (rElementalDofList.size() != 1) {
+                rElementalDofList.resize(1);
+            }
+            rElementalDofList[0] = GetGeometry()[0].pGetDof(TEMPERATURE);
+        }
+
+        void CalculateMassMatrix(
+            MatrixType& rMassMatrix,
+            const ProcessInfo& rCurrentProcessInfo) override
```

Comment:
:+1: for using the new interface!

(and note to myself to finish the transition)